sentry.client.config.js

```diff
@@ -19,9 +32,12 @@ Sentry.init({
   // don't record replays for generic sessions
   replaysSessionSampleRate: 0,
   // record replays for errors
-  replaysOnErrorSampleRate: 1.0,
+  replaysOnErrorSampleRate: 0.1,
   integrations: [new Sentry.Replay()],
 
   ignoreErrors: sentryOptions.ignoreErrors,
   denyUrls: sentryOptions.denyUrls,
+  // only allow thirdweb.com and thirdweb-preview.com domains
+  allowUrls: [/^(https:\/\/)((thirdweb\.com)|(.*\.thirdweb-preview\.com))$/i],
```

Comment:
@MananTank @assimovt can you guys re-check this regex?

should allow:
- https://thirdweb.com (production)
- https://*.thirdweb-preview.com (preview deployments)

Reply:
regex looks good, just thinking if url needs to end $, what if its thirdweb.com/some/page

Reply:
This regex only allows the home page ( and only without trailing /  )

<img width="710" alt="image" src="https://github.com/thirdweb-dev/dashboard/assets/22043396/cbb04f5e-4ff8-45b4-9352-aca81d4db0fb">

Reply:
This allows this, but shouldn't :

<img width="576" alt="image" src="https://github.com/thirdweb-dev/dashboard/assets/22043396/225bf1e3-452c-48f4-b963-6b55dd45682e">

Reply:
This allows this, but shouldn't :

<img width="563" alt="image" src="https://github.com/thirdweb-dev/dashboard/assets/22043396/b68e8ced-2fae-489a-992f-72cd8c320058">

Reply:
@MananTank can you check if they go by domain or if they check the full URL?